### DISCUSSION TITLE
Image | Odroid M1S

### DISCUSSION THREAD

**Discussion by in5tall** [MichaIng/DietPi#7312] [Image Request]
Hello, 
can you please build a image for the odroid m1s?

Kindly regards from erfurt

**Comment by Holli-x** [comment]
Hi, 
I made a tutorial to get Dietpi installed on the M1S:

[https://dietpi.com/forum/t/dietpi-for-odroid-m1s/23253](https://dietpi.com/forum/t/dietpi-for-odroid-m1s/23253)

**Comment by MichaIng** [comment]
PR up: #8229

**Reply by MichaIng** [reply]
Test images can be found here: https://dietpi.com/downloads/images/testing/
